rct229/ruletest_engine/ruletest_jsons/ashrae9012022/ENV/rule_5_44.json

```diff
@@ -0,0 +1,148 @@
+{
+    "rule-5-44-a": {
```

Comment:
This case failed because Surface 1 is Unregulated type. Can you double-check on this?

Reply:
Case (b)  failed as we expected, but the surface 1 was also unregulated type. Should we also fix this in case (b)?